docs/architecture/tls13-support.md

```diff
@@ -140,12 +140,11 @@ MVP definition
   | MBEDTLS_SSL_DEBUG_ALL                    | no      |
   | MBEDTLS_SSL_ENCRYPT_THEN_MAC             | n/a     |
   | MBEDTLS_SSL_EXTENDED_MASTER_SECRET       | n/a     |
-  | MBEDTLS_SSL_KEEP_PEER_CERTIFICATE        | no      |
+  | MBEDTLS_SSL_KEEP_PEER_CERTIFICATE        | no (1)  |
   | MBEDTLS_SSL_RENEGOTIATION                | n/a     |
   | MBEDTLS_SSL_MAX_FRAGMENT_LENGTH          | no      |
   |                                          |         |
   | MBEDTLS_SSL_SESSION_TICKETS              | no      |
-  | MBEDTLS_SSL_EXPORT_KEYS                  | no (1)  |
```

Comment:
Because of

```
commit 5902cd64e2aa1e3e6be6fde498fa5b3d01fbe4bf
Author: Andrzej Kurek <andrzej.kurek@arm.com>
Date:   Tue Sep 28 10:00:32 2021 -0400

    Remove MBEDTLS_SSL_EXPORT_KEYS, making it always on
    This option only gated an ability to set a callback,
    but was deemed unnecessary as it was yet another define to
    remember when writing tests, or test configurations. Fixes #4653.
    Signed-off-by: Andrzej Kurek <andrzej.kurek@arm.com>
```

in PR #4989

(and then I didn't have to renumber the notes, so this diff is smaller)